docs/about-ensembl/legal-and-privacy/legal-and-privacy.md

```diff
@@ -1,8 +1,14 @@
 ---
-title: Legal and privacy issues of Ensembl use
-description: Legal and privacy issues of Ensembl use
+slug: legal-and-privacy
+title: Use of Ensembl content and data
+description:
```

Comment:
Didn't we want to have description as a mandatory field? I thought we did.

Reply:
Sorry, I forgot about that! I should read our documentation documentation 😀

Reply:
Plus I should probably rename this page, since it doesn't in itself cover privacy